# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../qtcore/hotfix-pkgconfig.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./tools/qdbus/src/src.pro~	2006-11-27 18:47:43.000000000 +0100
+++ ./tools/qdbus/src/src.pro	2007-02-28 20:04:24.000000000 +0100
@@ -39,6 +39,7 @@
    QMAKE_PKGCONFIG_INCDIR = $$[QT_INSTALL_HEADERS]
    QMAKE_PKGCONFIG_DESCRIPTION = Qt DBus module
    QMAKE_PKGCONFIG_NAME = QtDBus
+   QMAKE_PKGCONFIG_DESTDIR = pkgconfig
 }
 
 win32 {
@@ -89,7 +90,7 @@
 INSTALL_HEADERS = $$PUB_HEADERS
 targ_headers.files = $$INSTALL_HEADERS
 targ_headers.path = $$[QT_INSTALL_HEADERS]/QtDBus
-targ_pkgconfig.path = $$[QT_INSTALL_LIBS]/pkgconfig
+targ_pkgconfig.path = pkgconfig
 target.path = $$[QT_INSTALL_LIBS]
 INSTALLS += target targ_headers targ_pkgconfig
 
--- ./tools/qtestlib/src/src.pro.orig	2007-03-02 01:31:58.000000000 +0100
+++ ./tools/qtestlib/src/src.pro	2007-03-03 13:55:27.000000000 +0100
@@ -34,6 +34,7 @@
    QMAKE_PKGCONFIG_INCDIR = $$[QT_INSTALL_HEADERS]
    QMAKE_PKGCONFIG_DESCRIPTION = Qt Unit Testing Library
    QMAKE_PKGCONFIG_NAME = QtTest
+   QMAKE_PKGCONFIG_DESTDIR = pkgconfig
 }
 
 
@@ -75,6 +76,7 @@
 embedded:QMAKE_CXXFLAGS+=-fno-rtti
 
 target.path=$$[QT_INSTALL_LIBS]
+target_pkgconfig.path = pkgconfig
 INSTALLS        += target
 
 qt_install_headers {
